inc/dbmysql.class.php

```diff
@@ -1364,6 +1364,24 @@ function($idx) { return rtrim($idx, ','); },
          trim($structure)
       );
 
+      // SHOW CREATE TABLE does not display nullable field (only on VARCHAR and TEXT typed fields)
+      // we have to handle 'NULL' manually on these type of fields
+      $structure = preg_replace(
+        "/ (VARCHAR\(\d+\))\s+NULL/i",
+        " $1",
+        $structure
+      );
+      $structure = preg_replace(
+        "/ (TEXT)\s+NULL/i",
+        " $1",
+        $structure
+      );
+      $structure = preg_replace(
+        "/ (INT)\s+NULL/i",
+        " $1",
+        $structure
+      );
+
```

Comment:
There are only a few occurences of this in the installation file. If all supported DB servers/versions are acting the same, I would prefer to directly remove this extra NULL in the `glpi-empty.sql` file.
MySQL doc says: `If neither NULL nor NOT NULL is specified, the column is treated as though NULL had been specified.`

Reply:
I agree with that. It would be less code to maintain.
I'll do it.